src/system/BlocksModule/composer.json

```diff
@@ -25,23 +25,18 @@
             "core-compatibility": ">=3.0.0",
             "displayname": "Blocks Module",
             "url": "blocks",
```

Comment:
I don't think `"url"` is valid or used anymore

Reply:
it is still valid and used